src/pdb/mod.rs

```diff
@@ -353,11 +356,33 @@ impl BinWrite for Page {
 
         // Row Groups
         let mut relative_row_offset: u64 = Self::HEADER_SIZE.into();
-        for row_group in &self.row_groups {
+        let total_rows: u16 = if self.num_rows_large > self.num_rows_small.into()
+            && self.num_rows_large != 0x1fff
+        {
+            self.num_rows_large
+        } else {
+            self.num_rows_small.into()
+        };
+
+        let num_groups: u16 = if total_rows > 0 {
+            (total_rows - 1) / RowGroup::MAX_ROW_COUNT + 1
+        } else {
+            0
+        };
```

Comment:
isn't this just [`u16::div_ceil`](https://doc.rust-lang.org/std/primitive.u16.html#method.div_ceil)? (please double check).